common/services/prismic/parsers.js

```diff
@@ -544,6 +547,39 @@ export function parseOnThisPage(fragment: PrismicFragment[]): Link[] {
     });
 }
 
+export function parseMediaObjectLinks(
+  fragment: PrismicFragment[]
+): Array<MediaObjectType> {
+  const cannotParse = 'could_not_parse';
+  // filter media_object
+  const filteredMediaObject = fragment.filter(mediaObject => {
+    if (mediaObject.content?.type === 'media_object') {
+      return true;
+    }
+  });
+  // attempt to parse data
+  const parseMediaObjects = filteredMediaObject.map(mediaObject => {
+    try {
+      const { title, text, image } = mediaObject.content.data;
+      return {
+        id: mediaObject.content.id,
```

Comment:
Assuming this can be `undefined` if the editors haven't put it in?

Reply:
this is also generated

Reply:
I didn't know this - thanks for pointing it out.